tests/Unit/Mutators/HexBinaryMutatorTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Weebly\Mutate\Mutators;
+
+use PHPUnit\Framework\TestCase;
+
+class HexBinaryMutatorTest extends TestCase
+{
+    /**
+     * @param string $hex
+     * @param string $expected
+     * @dataProvider hexProvider
+     */
+    public function testSerialize($hex, $expected)
+    {
+        $this->assertEquals($expected, (new HexBinaryMutator())->serializeAttribute($hex));
+    }
+
+    /**
+     * @param string $expected
+     * @param string $binary
+     * @dataProvider hexProvider
+     */
+    public function testUnserialize($expected, $binary)
+    {
+        $this->assertEquals($expected, (new HexBinaryMutator())->unserializeAttribute($binary));
+    }
+
+    /**
+     * @return \Iterator
+     */
+    public function hexProvider()
+    {
+        for ($i = 0; $i < 10; $i++) {
+            $bytes = random_bytes(20);
+            yield [bin2hex($bytes), $bytes];
```

Comment:
We should find the hard coded tests cases that will pass/fail instead of a for loop and a random value, otherwise this test should be marked as flaky.